beetle_tester.py

```diff
@@ -0,0 +1,45 @@
+import arcade
+from beetles import Beetle
+from teams import Team
+from team_color import TeamColor
+
+# Constants
+SCREEN_WIDTH = 1280
+SCREEN_HEIGHT = 720
+SCREEN_TITLE = "Welcome to the Ouch, Motherfucker"
+
+class BeetleTestes(arcade.Window):
+    def __init__(self):
+
+        # Call the parent class constructor
+        super().__init__(SCREEN_WIDTH, SCREEN_HEIGHT, SCREEN_TITLE)
+
+        self.background = None
+        self.green_team = None
+        self.red_team = None
+
+        arcade.set_background_color(arcade.color.WHITE)
+
+    def setup(self):
+        self.background = arcade.load_texture("Assets/Images/octagon.png")
+
+        self.green_team = Team(TeamColor.GREEN, 320, 260)
+        self.green_team.set_up_team()
+
+        self.red_team = Team(TeamColor.RED, 960, 260)
+        self.red_team.set_up_team()
```

Comment:
Might be worth adding a TODO that we need to replace this with the tester set-up, which will likely be different from the game set-up.